src/components/BankDetails/RecipientDetails/RecipientDetailsForm/RecipientDetailsForm.tsx

```diff
@@ -123,31 +125,44 @@ export default function RecipientDetailsForm({
         const labelRequired = f.required ? true : undefined;
         if (f.type === "select") {
           return (
-            <div key={f.key} className="">
-              <Label required={labelRequired} htmlFor={f.key} className="mb-1">
+            <div key={f.key} className="field">
+              <Label required={labelRequired} htmlFor={f.key}>
                 {f.name}
               </Label>
-              <select
```

Comment:
The reason I opted for creating a shared "uncontrolled" selector component instead of just manipulating native `select`'s CSS is that we already have a shared [Selector](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.9.2/src/components/Selector/Selector.tsx) component with the desired styles and behaviors defined, so having additional ad hoc implementations of select, with their own CSS, would increase maintenance cost.

You may be asking "but hey, you say you wanted to avoid ad hoc implementations, but you created yet a new Selector implementation!" and you'd be right, I did. There are 2 reasons for this:
1. This "uncontrolled" Selector can be used in places where we want to use a selector without a surrounding `FormProvider`, like here in `BankDetails` (for an example of `FormProvider` usage see [ContactDetails/index.tsx](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.9.2/src/pages/Registration/Steps/ContactDetails/index.tsx#L47-L49)).
2. The current `Selector.tsx` can be rewritten to use this "uncontrolled" selector as its nested component (as they share all of the styles and behavior) - that way webapp-wide Selector changes can be done by manipulating _one single component_.
The way this would be done is that:
   - `SelectorUncontrolled` would contain the UI and the behavior
   - `Selector` would set up the `react-hook-form` wires and pass them to the actual selector.
I left this for a separate Linear ticket / PR, as the change would require thorough testing, so it's outside the scope of this PR.
P.S. I'm open to name suggestions for these components, I find `SelectorUncontrolled` too long winded, but hadn't though of any better (shorter) name.
3. The "uncontrolled" Selector can in the future be used in places where we only need a plain (yet aligned) selector component.